student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,6 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+   // TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+ TEST_ASSERT_EQUAL_STRING_MESSAGE(malloc_username_from_conf_file(), my_username(), "failed match");
```

Comment:
@yogesh-1303 you should free the string malloced from `malloc_username_from_conf_file` here.